modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceClientCredentialTestCase.java

```diff
@@ -47,19 +50,35 @@
 	private String accessToken;
 	private String consumerKey;
 	private String consumerSecret;
+	private String username;
+	private String userPassword;
 
 	private CloseableHttpClient client;
 
+	@DataProvider(name = "configProvider")
+	public static Object[][] configProvider() {
+		return new Object[][]{
+				{TestUserMode.SUPER_TENANT_ADMIN},
```

Comment:
i think line number 61 and 62 needs to be formatted. please check

Reply:
Fixed 947bac0